service/index/writer.go

```diff
@@ -116,7 +116,7 @@ func (w *Writer) Transactions(blockID flow.Identifier, collections []flow.LightC
 	for _, collection := range collections {
 		err := w.db.Update(storage.SaveCollection(collection))
 		if err != nil {
-			return err
+			return fmt.Errorf("unable to save collection: %w", err)
```

Comment:
Didn't the PR say that we stick to `could not` for everything? `could not save collection: %w", err`?